korim/src/commonMain/kotlin/com/soywiz/korim/format/ImageFrameLayer.kt

```diff
@@ -42,6 +43,7 @@ open class ImageFrameLayer constructor(
     val height get() = slice.height
     val area: Int get() = slice.area
     val bitmap: Bitmap get() {
+        if (returnBitmapInPlace) return slice.bmp
         if (_bitmap == null) _bitmap = slice.extract()
```

Comment:
Maybe instead of propagating the `returnBitmapInPlace` we can optimize this.
What if we check if the bounds of the slice are the whole bitmap, and if that's the case, we return the slice.bmp without extracting? That way it will work, be faster and will simplify by not having to propagate or have the returnBitmapInPlace property

Reply:
done